setup.py

```diff
@@ -178,7 +189,7 @@ def create_dir_symlink(src, dest):
 cuda_version = "0.0"
 # Set hip_version to 0.0 if cpu-only
 hip_version = "0.0"
-if torch.version.cuda is not None:
+if torch_available and torch.version.cuda is not None:
     cuda_version = ".".join(torch.version.cuda.split('.')[:2])
 if torch.version.hip is not None:
```

Comment:
@rraminen I think this condition also needs to consider `torch_available` based on the recent changes. Essentially, they seem to be checking for torch availability in multiple places now, and we need to make sure it's also done for any ROCm-specific checks such as these.

Reply:
@jithunnair-amd, made the change.